admissionController/certs/ca.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
why this certificate required?

Reply:
This is required since the communication with k8s server happens via https.